content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+                    - name: LDAP_TLS
+                      value: "false"
+                    - name: LDAP_REPLICATION
+                      value: "false"
+                    - name: KEEP_EXISTING_CONFIG
+                      value: "false"
+                    - name: LDAP_REMOVE_CONFIG_AFTER_SETUP
+                      value: "true"
+              volumes:
+                - name: ldap-config
+                  emptyDir: {}
+          volumeClaimTemplates:
+              - metadata:
+                  name: ldap-data
+                spec:
+                  accessModes: [ "ReadWriteOnce" ]
+                  resources:
+                    requests:
+                      storage: 10Gi
+        
+        ---
+        
+        apiVersion: v1
+        kind: Service
+        metadata:
+          labels:
+            app: phpldapadmin
+          name: phpldapadmin-service
+          namespace: kubeflow
+        spec:
+          type: ClusterIP
+          ports:
+            - port: 80
+          selector:
+            app: phpldapadmin
+        
+        ---
+        
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: phpldapadmin
+          namespace: kubeflow
+          labels:
+            app: phpldapadmin
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: phpldapadmin
+          template:
+            metadata:
+              labels:
+                app: phpldapadmin
+            spec:
+              containers:
+                - name: phpldapadmin
+                  image: osixia/phpldapadmin:0.8.0
+                  ports:
+                    - name: http-server
+                      containerPort: 80
+                  env:
+                    - name: PHPLDAPADMIN_HTTPS
+                      value: "false"
+                    - name: PHPLDAPADMIN_LDAP_HOSTS
+                      value : "#PYTHON2BASH:[{'ldap-service.kubeflow.svc.cluster.local': [{'server': [{'tls': False}]},{'login': [        {'bind_id': 'cn=admin,dc=arrikto,dc=com'}]}]}]"
+        {{< /highlight >}}
+        
+        </details>
+        
+        
+    2. Seed the LDAP database with new entries.
+        
+        ```bash
+        kubectl exec -it -n kubeflow ldap-0 -- bash
+        ldapadd -x -D "cn=admin,dc=arrikto,dc=com" -W
+        # Enter password "admin".
+        # Press Ctrl+D to complete after pasting the snippet below.
+        ```
+        
+        <details>
+        
+        <summary>LDAP Seed Users and Groups</summary>
+        ```ldif
+        # This already exists.
+        # If it doesn't, uncomment this.
+        #dn: dc=arrikto,dc=com
+        #objectClass: dcObject
+        #objectClass: organization
+        #o: Arrikto
+        #dc: arrikto
+        
+        dn: ou=People,dc=arrikto,dc=com
+        objectClass: organizationalUnit
+        ou: People
+        
+        dn: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
```

Comment:
Following on from my earlier comment, this definitely looks like example data populating an LDAP database. It's best to anonymize the data by using names like "John Doe", and to explain above the snippet that the names, org, etc are examples.

Reply:
Changed arrikto to example and replaced me and Vangelis with fake names.